Anguandia/quickcredit
# Add user view repayments' history from UI feature

#### What does this PR do?

  Enable browser to send user request to view specific loans' repayment
  history to the server and render the response returned by the server

#### Description of Task to be completed?

  - Add the option of viewing repayments to details function
  - Dynamically modify detail page title to accommodate loan repayment
  history display(detail function in the main script file)
  - Selectively display only repayment logs if the request is for repayments
  history
  - Fix issue of non-unique ID balance on detail page js include
  - Anchor view repayment history menu item to loans list
  page(client.html) in the general template(template.js)
  - Add wait status display to the detail page title and subtitle sections
  to show progress as content gets fetched from the server
  - Remove commented-out sections of the client page code

#### How should this be manually tested?

  - Signup with one client and one admin account
  - Create a loan application from the client account
  - Switch to the admin account and verify the user that created the
  loan
  - Approve the loan created and post some repayments(debit loan) while
  still in the admin account
  - Signout from admin and into the user account and view repayment history

#### What are the relevant pivotal tracker stories?

  [finishes #166778338]
